packages/gatsby/src/utils/node-manifest.ts

```diff
@@ -363,7 +389,16 @@ export async function processNodeManifests(): Promise<Map<
 
   const processNodeManifestQueue = fastq(processNodeManifestTask, 25)
 
-  for (const manifest of nodeManifests) {
+  if (totalManifests > NODE_MANIFEST_FILE_LIMIT) {
+    nodeManifests = [...nodeManifests]
+    nodeManifests.sort(nodeManifestSortComparerAscendingCreatedAt)
```

Comment:
This is just a nit 😄 feel free to ignore
```suggestion
    nodeManifests.sort(nodeManifestSortComparerAscendingUpdatedAt)
```

Reply:
Naw, is completely valid! :+1: